src/components/VaultRow/lib/VaultUserBalance.tsx

```diff
@@ -2,14 +2,15 @@ import { FC } from "react"
 
 import clsxm from "@/lib/clsxm"
 import { VaultProps } from "@/lib/types"
-import { useTokenOrNativeBalance } from "@/hooks"
+import { useTokenOrNativeBalance, useTokenPriceUsd } from "@/hooks"
 
 import { AssetBalance, AssetBalanceUsd } from "@/components/Asset"
 
 export const VaultUserBalance: FC<VaultProps> = (props) => {
   const { data: balance } = useTokenOrNativeBalance({
     address: props.vaultAddress,
   })
+  const tokenPriceUsd = useTokenPriceUsd({ asset: props.asset })
```

Comment:
The concentrator has a different way of getting the price that's why I externalize the price method here